doc/design/OrionMongoSink.md

```diff
@@ -75,7 +82,6 @@ NOTE: the results for the three different data models (<i>collection-per-service
 | type | yes | N/A | com.telefonica.iot.cygnus.sinks.OrionMongoSink |
 | channel | yes | N/A |
 | mongo_hosts | no | localhost:27017 | FQDN/IP:port where the MongoDB server runs (standalone case) or comma-separated list of FQDN/IP:port pairs where the MongoDB replica set members run
-| mongo\_replica\_set | no | <i>empty</i> | Replica set name, not necessary in standalone case
```

Comment:
What "Should not be public" (in comments column) means? Maybe it should be elaborated in the comments.